benchmark/Gemm_vs_ormqr.cc

```diff
@@ -36,8 +36,8 @@ test_speed(int64_t m, int64_t n, int64_t runs, RandBLAS::RNGState<RNG> const_sta
         auto state = const_state;
 
         RandLAPACK::gen::mat_gen_info<double> m_info(m, n, RandLAPACK::gen::gaussian);
-        RandLAPACK::gen::mat_gen<double, r123::Philox4x32>(m_info, A, state);
-        RandLAPACK::gen::mat_gen<double, r123::Philox4x32>(m_info, B1, state);
+        RandLAPACK::gen::mat_gen(m_info, A, state);
+        RandLAPACK::gen::mat_gen(m_info, B1, state);
```

Comment:
I think you should be able to omit these template arguments without having made any changes to ``RandLAPACK::gen::mat_gen``. Am I wrong?

Reply:
Yes, but i also did get rid of the RNG template argument in `RandLAPACK::gen::mat_gen` because it was not needed.